### PR TITLE
Fix issues found by snyk-deps

### PR DESCRIPTION
**Pull request by bear-redhat**

### CODE REVIEW

File: .snyk [hunk 1]
```diff
@@ -2,4 +2,48 @@ exclude:
   global:
     - "vendor/**"
     - "**/*_test.go"
-
+ignore:
+  'snyk:lic:golang:github.com:hashicorp:vault:api:MPL-2.0':
```
Comment (droslean) [inline .snyk:6]: Can we do something like `snyk:lic:golang:github.com:*:MPL-2.0` here?

Comment (bear-redhat) [reply .snyk:6]: Tried, not working

**Pull request merged**